explorer/site-config/hca-dcp/dev/projectViewModelBuilder.ts

```diff
@@ -431,6 +432,8 @@ export const projectsToSpeciesColumn = (
       project.donorOrganisms.flatMap((orgnanism) => orgnanism.genusSpecies)
     ),
     customColor: "ink",
+    // TODO(cc) resolve with ticket Update project details component to match refined mocks #94
```

Comment:
I accidentally removed theeslint-disable sonarjs/no-duplicate-string comment ... then put it back... but kept this comment in anyway!